core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -461,6 +461,36 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  def updateBillingProjectBillingAccount(billingProjectName: RawlsBillingProjectName, updateAccountRequest: UpdateRawlsBillingAccountRequest): Future[Unit] = {
+    requireProjectAction(billingProjectName, SamBillingProjectActions.updateBillingAccount) {
```

Comment:
From what I can tell, this permission check is not present in the v2_workspaces branch. Should I just file a Jira for it to be added?